pkgs/tools/security/hashcash/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, openssl }:
+
+stdenv.mkDerivation rec {
+  name = "hashcash-${version}";
+  version = "1.22";
+
+  buildInputs = [ openssl ];
+
+  src = fetchurl {
+      url = "http://www.hashcash.org/source/hashcash-1.22.tgz";
+      sha256 = "15kqaimwb2y8wvzpn73021bvay9mz1gqqfc40gk4hj6f84nz34h1";
+  };
+
+  makeFlags = "generic-openssl LIBCRYPTO=-lcrypto";
```

Comment:
Please split the arguments into a list.

Reply:
Done in 6b8c09bffca82c1373c38586cac2032088f0e5bf